web/styles/subscriptions.css

```diff
@@ -519,12 +478,8 @@ h4.user_group_setting_subsection_title {
         cursor: pointer;
     }
 
-    .left,
-    .right {
-        position: relative;
-        display: inline-block;
-        vertical-align: top;
-        margin: 0 -2px;
+    .two-pane-settings-body {
+        display: flex;
```

Comment:
Why not grid here? Since this is the parent of `.left` and `.right`, I think this would be better suited to grid, and allowing us to use `transform` while also jettisoning the lingering `position: absolute` values hanging out even after the other work here.